tasks/main.yml

```diff
@@ -52,6 +53,12 @@
   tags:
     - mapnik
 
+- name: Replace python version in configure file
+  lineinfile:
+    regexp: "^:"
+    line: ": ${PYTHON:=python3}"
```

Comment:
@FrankApiyo do you think it'd be worthwhile fetching `python3` from a variable?

Reply:
added a variable!